.github/workflows/unit-tests.yml

```diff
@@ -430,3 +430,26 @@ jobs:
           # do not import pandas from the checked out repo
           cd ..
           python -c 'import pandas as pd; pd.test(extra_args=["-m not clipboard and not single_cpu and not slow and not network and not db"])'
+
+  numpy-1.26-test:
```

Comment:
We'll want to create a new `yaml` file of dependencies and add a job in the `includes` section above